testdata/test1/hierarchy.lst

```diff
@@ -0,0 +1,8 @@
+# This file has various combinations of comments and ways of specifying the path
+# Next line are the initial values
+../default
+../yaml # different yaml files
+../json/
```

Comment:
```suggestion
../json/
```
Does this need the ending slash?

Reply:
Yes I purposely have different variations in there, in case we mess something up. Let me add a comment